slowmovie.py

```diff
@@ -50,23 +58,27 @@ def generate_frame(in_filename, out_filename, time):
         .run(capture_stdout=True, capture_stderr=True)
     )
 
+
 def check_vid(value):
     if not os.path.isfile(value):
         raise configargparse.ArgumentTypeError("File '%s' does not exist" % value)
     if not supported_filetype(value):
         raise configargparse.ArgumentTypeError(f"File '{file}' should be a file with one of the following supported extensions: {', '.join(fileTypes)}")
     return value
 
+
 def check_dir(value):
     if os.path.isdir(value):
         return value
     else:
         raise configargparse.ArgumentTypeError("Directory '%s' does not exist" % value)
 
+
```

Comment:
What's up with all these extra blank lines?

Reply:
I'm guessing you mean between the function defintions? PEP8 specification is 2 spaces between function and class definitions.

Reply:
Oh, okay.